app/src/ui/changes/changes-list.tsx

```diff
@@ -1012,6 +1012,7 @@ export class ChangesList extends React.Component<
             >
               <Checkbox
                 ref={this.includeAllCheckBoxRef}
+                className="changes-list-check-all"
```

Comment:
This lets me keep the unique check all box styles from interfering with the generic check box styles in the filter popover.